collectivites/static/js/map-nav.js

```diff
@@ -0,0 +1,36 @@
+const defaultCenter = [47, 2]
+function MapNav () {
+    riot.observable(this)
+    let map
```

Comment:
Humm, U sure? We still need the reference in various function why only want to instantiate it in the `init` callback, no?